Makefile

```diff
@@ -27,40 +27,24 @@ PYTEST_ARGS ?=
 BUILD_IMAGES ?= 1
 
 ifdef KEEP_TEST_CONTAINER
-  DOCKER_RM_CONTAINER =
+  CONTAINER_RM =
 else
-  DOCKER_RM_CONTAINER = --rm
+  CONTAINER_RM = --rm
 endif
 
-# Let the user specify DOCKER at the CLI, otherwise try to autodetect a working podman or docker
-ifndef DOCKER
-  DOCKER := $(shell podman run --rm alpine echo podman 2> /dev/null)
-  ifndef DOCKER
-    DOCKER := $(shell docker run --rm alpine echo docker 2> /dev/null)
-    ifndef DOCKER
-      DUMMY := $(warning Many of the make targets require a working podman or docker.  Please install one of those and check that `podman run alpine echo hello` or `docker run alpine echo hello` work)
-    endif
-  endif
+# Let the user specify PODMAN at the CLI, otherwise try to autodetect a working podman
+ifndef PODMAN
+  PODMAN := $(shell podman run --rm alpine echo podman 2> /dev/null)
```

Comment:
I will let you decide on this, @SpyTec: 

I think it would be nice to output that warning if no `podman` was found during this check. The user will probably see some errors happening in later phases of the Makefile, but I think that keeping the warning message would be valuable.

Reply:
Done!